### ISSUE TITLE
Refactor: Create Reusable Function for API call

### ISSUE THREAD

**Issue by ash1eygrace** [ash1eygrace/ai-content#3]
Assign form fields to variables that'll pass through into the function for each call to API. The call requires: 

```
 openai.createCompletion("text-davinci-002", {
            prompt: `${formDataObj.examleVFormField}`,
            temperature: 0.85,
            max_tokens: 200,
            top_p: 1,
            frequency_penalty: 0,
            presence_penalty: 0,
        })
```

Set defaults on call and a condition that determine if a custom value is being passed through, else default values. 

Possible solution: 
https://reactjs.org/docs/hooks-rules.html
https://reactjs.org/docs/hooks-custom.html 
Example:  https://www.freecodecamp.org/news/how-to-create-react-hooks/

**Comment by ash1eygrace** [comment]
https://www.reddit.com/r/reactjs/comments/ug84ev/how_do_you_decide_when_to_create_a_custom_hook/?utm_source=share&utm_medium=ios_app&utm_name=iossmf

**Comment by ash1eygrace** [comment]
Working on this here: https://github.com/ash1eygrace/ai-content/compare/main...reusable_function?expand=1

**Comment by ash1eygrace** [comment]
To be considered: 

- https://www.youtube.com/watch?v=pI7m262dBCc&t=1s 
- https://en.wikipedia.org/wiki/Generalization 
- https://en.wikipedia.org/wiki/Abstraction_(computer_science) 
- https://www.youtube.com/watch?v=6V1sr0XV_Ng

**Comment by ash1eygrace** [comment]
Possible structure: 

1. reusable API call function
2. form component
3. card component

**Comment by ash1eygrace** [comment]
Solved in Refactor api function #15
